tracking-migration/src/main/java/io/customer/tracking/migration/queue/Queue.kt

```diff
@@ -37,21 +35,10 @@ internal class QueueImpl internal constructor(
         }
     }
 
-    internal fun runAsync() {
+    override fun runAsync() {
+        logger.debug("Starting migration queue runner...")
         CoroutineScope(dispatchersProvider.background).launch {
             run()
         }
     }
-
-    private fun processQueueStatus(queueStatus: QueueStatus) {
```

Comment:
I think we don't need it any more because we'll be calling `runAsync` directly from migration class